tasks/destroy-volumes.yml

```diff
@@ -5,6 +5,7 @@
     {{ item.name }}
     {{ item.pool }}
```

Comment:
is `pool` going to defined for network volumes?

Reply:
Good question, I definitely meant to check but forgot. I've added a default, so it is the same as during creation.